player/low-latency-streaming/js/script.js

```diff
@@ -8,9 +8,9 @@ var latencyDisplay = document.querySelector('#latency');
 var slider = document.querySelector('#targetLatencySlider');
 var targetLatencyDisplay = document.querySelector('#targetLatency');
 
-var targetLatency = 3;
+var targetLatency = 5;
```

Comment:
Please use this value in the player config (line 77) 🙂

Reply:
done!